cmd/ttn-lw-stack/commands/start.go

```diff
@@ -114,6 +118,37 @@ var (
 				return err
 			}
 
+			const (
+				interopClientConfRepoPath  = ""            // TODO: Define the location of interop client configuration repository.
```

Comment:
@htdvisser I would like to add flags for these, but it's not obvious for me how to do that in current implementation.
What is the expected way to add CLI flags to binaries in the current implementation?